packages/flame_isolate/example/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
How come the example licence is GPLv2 instead of MIT?

Reply:
Due to the beautiful worker asset. Added a small note about that in the README.